### DISCUSSION TITLE
No way to access some parts of the research outpost

### DISCUSSION THREAD

**Discussion by Gelleode** [FakeFishGames/Barotrauma#14156] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![image](https://github.com/FakeFishGames/Barotrauma/assets/48509532/b6255bf4-0383-4476-84be-c888118d8d4f)
![image](https://github.com/FakeFishGames/Barotrauma/assets/48509532/97bc5f17-8120-4d62-b6ce-4cf430cee558)
The hatch to access clowns above is non existant. I could hear their shoes there so i am sure that it is them above me besides there were a few clowns randomly beating my crew.

### Reproduction steps

1. Dock to research outpost
2. Find out you can't get clowned

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.7.0 (Summer Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! Do you happen to have the save file available, and could you send it here (note that you need to .zip it first since github doesn't support the .save file extension)? Since the outposts are randomly generated, it might be difficult for us to find an outpost where this issue occurs. Thanks and sorry for the inconvenience!

**Comment by Regalis11** [comment]
Found a seed where this happens, in level editor:
```
seed: 17
level: outpostlevel
outpost parameters: researchoutpost
submarine: barsuk
```

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/effeac99b463a54dd73e2a0d97b581a8bc924349 (commit in our private repo). Coming up in the next hotfix.
